CoreScriptsRoot/Modules/Server/ClientChat/MessageLabelCreator.lua

```diff
@@ -303,6 +327,46 @@ function methods:CreateSetCoreMessageLabel(valueTable)
 	return WrapIntoMessageObject(-1, BaseFrame, BaseMessage, Tweener, {}, nil, self.ObjectPool)
 end
 
+function methods:CreateMeCommandChannelEchoMessageLabel(messageData, echoChannel)
+	--// Cannot be destructive with messageData
+	local oldMessage = messageData.Message
+	local oldSpeaker = messageData.FromSpeaker
+	local oldChatColor = messageData.ExtraData.ChatColor
+
+	local tempMessage = messageData.FromSpeaker .. " " .. string.sub(oldMessage, 5)
+
+	local extraData = messageData.ExtraData or {}
+	local useFont = extraData.Font or Enum.Font.SourceSansBold
+	local useFontSize = extraData.FontSize or ChatSettings.ChatWindowTextSize
+
+	if not messageData.IsFiltered then
```

Comment:
I think just this part needs to be factored out when in module mode, everything else is pretty trivial